dashboard/app/models/lesson.rb

```diff
@@ -663,6 +663,9 @@ def next_level_for_lesson_extras(user)
   end
 
   def next_level_path_for_lesson_extras(user)
+    if script.show_unit_overview_between_lessons?(user)
```

Comment:
Will user ever be `nil`? Would we ever call this for a logged out user?

Reply:
You can get to Lesson Extras if they are available in the script and you're in a section where the teacher turned lesson extras on. If you are logged out and try to hit an /extras url we'll prompt you to sign in.
![Screen Shot 2021-12-01 at 11 25 23 AM](https://user-images.githubusercontent.com/12300669/144273266-fe16501f-cab7-4e81-9dde-51b64b157ab5.png)

But, yes, I'll safeguard from trying to check if the experiment is enabled when user is nil.

Reply:
Lesson extras will be available if they are available on the script and if a student's teacher turns them on for the section.  If you're logged out and try to hit an /extras url we prompt you to sign in. 
![Screen Shot 2021-11-30 at 12 43 06 PM](https://user-images.githubusercontent.com/12300669/144274803-4b0007c4-1826-4741-88d7-153d073f305e.png)

But, yes, I'll add a safeguard to ensure we don't try to check if the experiment is enabled when the user is `nil`.